src/notification.c

```diff
@@ -317,6 +317,8 @@ void notification_unref(struct notification *n)
         notification_private_free(n->priv);
 
         if (n->script_count > 0) {
+                for (int i = 0; i < n->script_count; i++)
+                        g_free(n->scripts[i]);
```

Comment:
I feel like there is a glib function to do this

Reply:
I didn't find it

Reply:
I didn't find it

Reply:
modified to use gstrfreev